docs/reference/supported-environments.md

```diff
@@ -156,3 +156,19 @@ You can also use newer versions of Desktop and Web Modeler with older versions o
 </TabItem>
 
 </Tabs>
+
+## Dependency maintenance policies
+
+Camunda provides [a standard support policy](https://camunda.com/release-policy/) of 18 months for a particular minor version from the date it is released.
+During this time, patches are regularly released containing security and bug fixes, some of which may come from dependency updates. Therefore, for the
+vast majority of dependencies Camunda _only_ applies patch updates.
+
+However, certain dependencies used by Camunda 8 may have a shorter maintenance policy than Camunda itself. Camunda may adopt a different update policy for these dependencies, as listed below.
+
+### Spring
+
+Many server side components, such as the **Camunda Orchestration Cluster**, are Spring Boot applications, and leverage Spring Boot (and the wider Spring ecosystem) to execute fundamental functionality such as application configuration, REST infrastructure (including security), production ready features, etc.
+
+However, Spring has a shorter maintenance window than Camunda for its open-source software (OSS) offering. Versions are only supported for 12 months, versus Camunda's 18 months. To circumvent this, server-side components of Camunda 8 patch releases also update Spring (and Spring Boot) minor versions, such that the latest patch release of these components uses a supported Spring version.
```

Comment:
You could link here to the support policy documentation for [Spring](https://spring.io/projects/spring-framework#support) and [Spring Boot](https://spring.io/projects/spring-boot#support). In that sense, 12 months for Spring is not fully accurate (it's Spring Boot, Spring Framework tends to have longer support durations). Maybe you can clarify that or make the statement more generic (like "Spring and Spring Boot minor versions can go out of maintenance during the maintenance window of a Camunda minor release").

Reply:
@npepinpe Can we explicitly limit it to the orchestration cluster?

Reply:
Spring support is a little hard to explain, since it's tied to the Spring Boot version. But I can keep it to Spring Boot for now.

> All project support dates are mapped to the version of Spring Boot that either manages it or it depends upon (the support dates for a given project will match those of the Spring Boot version it’s mapped to).

Linking is a good idea :+1: